### PR TITLE
Three button menu changes

### PR DESCRIPTION
**Pull request by deeppandya**
Addresses : https://github.com/brave/browser-android-tabs/issues/1718

### CODE REVIEW

**Comment by SergeyZhukovsky** [comment]
Are we sure we want to use the current icon for share? Chrome uses that one
ic_share_white_24dp
cc @anthonypkeane @jamesmudgett

File: chrome/android/java/strings/android_chrome_strings.grd [hunk 1]
```diff
@@ -2129,6 +2129,9 @@ To change this setting, <ph name="BEGIN_LINK">&lt;resetlink&gt;</ph>reset sync<p
       <message name="IDS_ACCESSIBILITY_MENU_INFO" desc="Content description for the button which opens the page info popup, which displays security information and permissions for the current page">
         View site information
       </message>
+      <message name="IDS_ACCESSIBILITY_MENU_SHARE" desc="Content description for the button which opens the page info popup, which displays security information and permissions for the current page">
+        Share site
```
Comment (SergeyZhukovsky) [inline chrome/android/java/strings/android_chrome_strings.grd]: Where did you get that string? I don't see it in the original issue. @anthonypkeane what should it be here, it is when you press a long click on it.

Comment (anthonypkeane) [reply chrome/android/java/strings/android_chrome_strings.grd]: I've update the text we should add for long press on https://github.com/brave/browser-android-tabs/issues/1718

**Comment by deeppandya** [comment]
Closing this PR as it was created from branch of base-75.0.3770.89. Creating a new PR from the branch of base-75.0.3770.101.

**Pull request closed without merging**